apps/wallet/src/index.tsx

```diff
@@ -14,24 +15,33 @@ import { linking, navigationRef } from 'utils/navigation';
 export const AppStack = () => {
 	const modalContainerRef = useRef<View>(null);
 	const theme = useSnapshot(themeState);
+	const { isMobileDisplay } = useSnapshot(appState);
 	const hydrate = useNavigationHydrate();
 
 	useNotifications();
 	useEffect(() => modalActions.setContainerRef(modalContainerRef), []);
 
 	return (
 		<SafeAreaProvider>
-			<View style={styles.container} ref={modalContainerRef}>
-				<NavigationContainer
-					ref={navigationRef}
-					theme={theme}
-					linking={linking}
-					onReady={hydrate.onNavigationReady}
-					onStateChange={hydrate.onNavigationStateChange}
+			<View style={[styles.container, !isMobileDisplay && styles.centerLayout]}>
+				<View
+					style={[
+						styles.innerLayout,
+						!isMobileDisplay && styles.extensionLayout,
+					]}
```

Comment:
for long block like this, if possible promote it as a variable outside XML/Markup block, e.g:
```typescript
const innerStyle: StyleProp<ViewStyle> = [
  styles.innerLayout,
  !isMobileDisplay && styles.extensionLayout,
];

<View ...>
  <View style={innerStyle}>
</View>
```